openedx/core/djangoapps/waffle_utils/testutils.py

```diff
@@ -2,66 +2,17 @@
 Test utilities for waffle utilities.
 """
 
-
-from waffle.testutils import override_flag
+# Import from edx-toggles to preserve import paths
+# pylint: disable=unused-import
+from edx_toggles.toggles.testutils import override_waffle_flag
```

Comment:
Should we make a lightweight wrapper with a deprecation warning so we can get rid of this?

Reply:
Again, I'm not a big fan of adding yet another deprecation warning that will print a lot of output. Can we add this as a future action item and do this in a separate PR?

Reply:
This can definitely be a part of a future iteration.  Maybe a comment?:
```
# TODO: Deprecate and remove.
```

Reply:
:ok_hand: